newIDE/app/src/ExportAndShare/ShareDialog/PublishHome.js

```diff
@@ -313,7 +313,7 @@ const PublishHome = ({
           <Trans>You must be connected to use online export services.</Trans>
         </AlertMessage>
       )}
-      {!chosenSection && !showOnlineWebExporterOnly && (
+      {!chosenSection && (
```

Comment:
Theoretically not needed, but still changed so we can decide in the future to show the "gd.games" button before the export screen.